drugex/training/scorers/rocs_openeye.py

```diff
@@ -0,0 +1,364 @@
+import os
```

Comment:
```suggestion
"""OpenEye-ROCS based ROCS scorer."""
import os
```

For consistency with the other ROCS scorers